Tests/XcbeautifyLibTests/RendererTests/GitHubActionsRendererTests.swift

```diff
@@ -510,6 +515,14 @@ final class GitHubActionsRendererTests: XCTestCase {
         XCTAssertEqual(parser.outputType, .testCase)
         #endif
     }
+    
+    func testTestCaseSkipped() {
+        #if os(macOS)
+        let formatted = logFormatted("Test Case '-[SomeTests testName]' skipped (0.004 seconds).")
+        XCTAssertEqual(formatted, "::notice ::    testName (0.004 seconds)")
+        XCTAssertEqual(parser.outputType, .testCase)
```

Comment:
This value is attached to the type, so you can delete this assert.

Reply:
I don't really get this note - but removed - I initially put it there because all tests mirroring this one had that line.